app/views/admin/deploy_groups/show.html.erb

```diff
@@ -40,7 +40,16 @@
                     class: "btn btn-default"
         %>
         <%= link_to "Deploy All Projects", deploy_all_admin_deploy_group_path(@deploy_group), class: "btn btn-default", data: { method: "post" } %>
-      <% end %>
+        <%= link_to "Merge All Stages",
```

Comment:
sounds super dangerous and is not what is happening ... it finds stages that are only deploying to this deploy group and merges them into the template ...